types/chrome-remote-interface/chrome-remote-interface-tests.ts

```diff
@@ -0,0 +1,23 @@
+import CDP from "chrome-remote-interface";
```

Comment:
The chrome-remote-interface's readme says that JS imports look like `const CDP = require("CDP")`, which means that this line needs to use the Typescript equivalent `import CDP = require("CDP")`. And the export needs to be `export = CDP` not `export default CDP`.